bootstrap/api/transport.go

```diff
@@ -258,16 +258,16 @@ func encodeSecureRes(_ context.Context, w http.ResponseWriter, response interfac
 func encodeError(_ context.Context, err error, w http.ResponseWriter) {
```

Comment:
Yes I can open another PR for this. Similar to how processRequest was unified earlier in sdk

Reply:
It is repeating because not all the services have the same errors. However, if it's similar enough, which it is, it's worth extracting it to apiutils.